conf/locale/ar/LC_MESSAGES/django.po

```diff
@@ -5335,14 +5335,12 @@ msgstr ""
 #: themes/st-lutherx/lms/templates/student/edx_ace/passwordreset/email/body.html
 #: themes/st-normanx/lms/templates/student/edx_ace/passwordreset/email/body.html
 msgid "Change my Password"
-msgstr ""
-"تغيير كلمة المرور الخاصة بي"
+msgstr "تغيير كلمة المرور الخاصة بي"
```

Comment:
revert this. Its the same translation as before.

Reply:
Done